app/templates/app/project_detail.html

```diff
@@ -40,7 +40,13 @@ <h1 id="main-heading">{{ project.name }}</h1>
 
           <p class="col-lg-6">
               {% icon "date" %}
-              {% trans "End Date:" %} {{ project.end_date|date:"Y-m-d" }}
+              {% trans "End Date:" %}
+              
+              {% if project.end_date %}
+                {{ project.end_date|date:"Y-m-d"|default:"" }}
```

Comment:
Nitpick: please use four spaces for indenting as I believe most of the rest of the template code uses.

Reply:
It looks like (in this file at least) if statements are not indented at all 🤔